alf/utils/dist_utils.py

```diff
@@ -370,7 +370,11 @@ def __init__(self, loc, scale):
             loc (Tensor): mean of the distribution
             scale (Tensor): standard deviation. Should have same shape as ``loc``.
         """
-        super().__init__(td.Normal(loc, scale), reinterpreted_batch_ndims=1)
+        # set validate_args to False here to enable the construction of Normal
+        # distribution with zero scale.
```

Comment:
Is zero scale needed in any case?

Reply:
We have a test case for this:

https://github.com/HorizonRobotics/alf/blob/3ae0b511e63fb0bf9586712863b63267a43aaff4/alf/networks/projection_networks_test.py#L65-L86